direct_subset.py

```diff
@@ -42,16 +42,17 @@ def db_creator(db_type, source, dest):
         subsetter.prep_temp_dbs()
         subsetter.run_middle_out()
 
-        if "--no-constraints" not in sys.argv:
-            database.add_constraints()
 
         print("Beginning post subset SQL calls")
         start_time = time.time()
         for idx, sql in enumerate(config_reader.get_post_subset_sql()):
             print_progress(sql, idx+1, len(config_reader.get_post_subset_sql()))
             db_helper.run_query(sql, destination_dbc.get_db_connection())
         print("Completed post subset SQL calls in {}s".format(time.time()-start_time))
-
+        
+        print("Adding database constraints")
+        if "--no-constraints" not in sys.argv:
+            database.add_constraints()
```

Comment:
Just curious, but what was the intent around moving the database constraints later? While I'm sure it might be faster to run those post-subset SQL calls, it also means those calls will not be maintaining foreign key constraint checks at that time, so I'm hesitant to change this. What are your thoughts?

Reply:
Gotcha. I would say then it would be best to add another config point that makes this more obvious — a pre-constraints SQL section, and a post-constraints SQL section, just so users can know what they're getting into 😄. If you could add that and some documentation around it, it would definitely be appreciated and I'd accept it!